### ISSUE TITLE
TypeHinting issue.

### ISSUE THREAD

**Issue by HypertextPP** [friends-of-reactphp/mysql#38]
I have a very simple select query:
```
$dBase->query('SELECT * FROM `table` WHERE `column` = ?', function(QueryCommand $command, $conn) {
    if($command->hasError()) {
        $this->log('MySQL Error');
    } elseif(count($command->resultRows) < 1) {
										
    }
}, []);
```
However, for whatever reason, my IDE is unable to find `$resultRows`:
![Screenshot](https://user-images.githubusercontent.com/10724949/37896038-cfc7e6e6-30da-11e8-8960-69596851dabe.png)
Which makes sense as when I had a look at the class, neither was I.

If I am doing something wrong please tell me but if I am not, where is that field coming from?

**Comment by clue** [comment]
Yes, I can confirm that this is both correct and confusing :+1: PHP allows adding arbitrary attributes to objects and the `resultRows` attribute will automatically be added to this object.

This will be addressed with #4 soon-ish, which means we will also use a stricter interface for this :shipit: 

I hope this helps :+1:

**Comment by HypertextPP** [comment]
Cool, look forward to it! Thanks.
